ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/StreamedQueryResult.java

```diff
@@ -90,4 +93,9 @@
    */
   boolean isFailed();
 
+  boolean hasContinuationToken();
```

Comment:
`hasContinuationToken()` and `getContinuationToken()`only seem to be used in tests.  It would probably be best to avoid adding these to the public API and just caste the class to the impl version to expose them.

Reply:
I guess I was thinking of the original example like below we had where the user checks if there is a continuation token and then retries it, but I think it makes to sense to check that internally and hide the tokens logic from the user if possible.
```
  if (result.hasError() && result.hasContinuationToken()) {
    // Explicit retry or could store and try again later
    result.retry()
  }```

Reply:
I removed them from the public api and cast to the impl version in the tests